src/lib/viewers/controls/rotate/RotateControl.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import IconRotate24 from '../icons/IconRotate24';
+import './RotateControl.scss';
+
+export type Props = {
+    onRotateLeft: () => void;
+};
+
+export default function RotateControl({ onRotateLeft }: Props): JSX.Element {
+    return (
+        <div className="bp-RotateControl">
```

Comment:
I don't think this needs to be in its own group. Can just be the button on its own.

Reply:
Okay, will remove!